### PR TITLE
Prac 10 feedback

### PR DESCRIPTION
**Pull request by Quinn368**
@Ajeetha0701
@ChristineYamin
@ivy-naw-mth

### CODE REVIEW

File: prac_10/recursion.py
[the remark this answers is not in the document]
Comment (Quinn368) [reply prac_10/recursion.py:1]: Thanks~ But actually, I did not know that it was an extension at first, that's why I just did it. And when I checked for what to submit, it said only wiki and testing so... haha

[the remark this answers is not in the document]
Comment (ivy-naw-mth) [reply prac_10/recursion.py:1]: But I think it's a good thing to save your codes as you worked hard on it.

**Comment by Ajeetha0701** [comment]
you did an amazing job! the code is looking good!

File: prac_10/app.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+
+@app.route('/greet')
+@app.route('/greet/<name>')
+def greet(name=""):
+    return f"Hello {name}"
+
+
+if __name__ == '__main__':
+    app.run()
```
Comment (ChristineYamin) [inline prac_10/app.py]: Your coding is perfect.

File: prac_10/README.md [hunk 2]
```diff
@@ -1 +1,12 @@
...
+
+During the practical tasks, I got better at estimating how long things would take. I learned that when making a program, breaking big problems into smaller ones is really important, especially this can be utilized when facing unexpected issues.
+
+2. Regarding code reviews, what have you learned from both being reviewed and doing reviews?
+
+I found out that there are multiple ways to improve a program through reviews. Additionally, I learned a lot from the feedback and suggestions given by others.
+ 
+3. Regarding the practical tasks overall, what would you change if you were in charge of the subject? 
+
+If I were in charge of the subject, I would consider incorporating with the real-world projects to reinforce theoretical concepts and practical skills. Additionally, I would emphasize collaborative learning opportunities such as pair programming and group projects, to enhance teamwork, communication, and problem-solving skills among students.
```
Comment (ChristineYamin) [inline prac_10/README.md]: In ReadMe.md, you need to write the code review as the instruction asked to do.

**Pull request merged**